python/_restclient/README.md

```diff
@@ -1,13 +1,30 @@
 # HydroTools :: REST Client
 
-This subpackage contains classes and utilities suitable for quickly developing robust
-REST api interfaces. The primary feature of this subpackage is the generic
-`RestClient` class which offers sqlite database GET request caching, GET request
-backoff, and flexible url argument encoding. Immutable `Alias` and `AliasGroup`
-utility classes support common library development patterns aiding in the separation
-of backend and frontend code. See the [REST Client
-Documentation](https://noaa-owp.github.io/hydrotools/hydrotools._restclient.html) for
-a complete list and description of the currently available methods. To report bugs or
+Subpackage of objects and utilities suitable for quickly developing robust
+REST api libraries.
+
+Main features:
+
+- `RestClient`: a class which offers asynchronous performance via a convenient serial
+  wrapper, sqlite database GET request caching, GET request backoff, batch GET requests,
+  and flexible url argument encoding.
+
+- `Url`: Treat urls like to `pathlib.Path`s. Supports appending paths with `/` and
+  appending url query parameters with `+`.
+
+- `ClientSession`: Extension of
+  [`aiohttp_client_cache`](https://github.com/JWCook/aiohttp-client-cache) that adds
+  exponential backoff.
+
+- `Variadic`: Join list or tuple of objects on some delimiter. Useful when query
```

Comment:
I'm a bit confused by the use of `Variadic` here. I understand this implements a variadic process, but I don't think it's applicable to any variadic process. The code below just looks like a `string.join` with some extras. Does the method work on any `object`?

Reply:
It will work for any object that can be represented as a string. Admittedly the name might not be the most appropriate, do you have suggestions for an alternative name?